.changes/next-release/feature-AWSSDKforJavav2-8c45016.json

```diff
@@ -0,0 +1,6 @@
+{
+    "category": "AWS SDK for Java v2", 
+    "contributor": "", 
+    "type": "feature", 
+    "description": "Add methods to ProfileFileSupplier for aggregating"
```

Comment:
Since we haven't merged the APIs to mainline yet, you don't need a changelog entry for this

Reply:
Removed.